src/Article/Repository/ArticleRepository.php

```diff
@@ -111,10 +115,15 @@ public function getPaginatedArticleList(int $page, int $resultPerPage): Paginate
                 '=',
                 'ares_articles_comments.article_id'
             )->groupBy('ares_articles.id')
-            ->where('ares_articles.hidden', 0)
-            ->orderBy('ares_articles.id', 'DESC')
+            ->orderBy('ares_articles.id', $order)
             ->addRelation('user');
 
+            if(!$showHidden)
```

Comment:
Please write lines 121 - 126 like this:

```
if (!$showHidden)  {
    $searchCriteria = $searchCriteria->where('ares_articles.hidden', 0); <-- // also do we need the ares_articles? since the   tablename is already in the context
}
```

also remove multiple line spacing like on 126 / 126, there should be only one line space between

Reply:
About the 'ares_articles.?' I just followed same pattern as on top, where that's also used